src/app/features/characters/components/random-character/random-character.component.ts

```diff
@@ -1,16 +1,51 @@
-import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
```

Comment:
do you need the whole `CommonModule` here?

Reply:
No, I've deleted it